README.md

```diff
@@ -502,6 +502,7 @@ class App extends React.Component<{
   }
 }
 ```
+Notice, if your TypeScript version before 2.7, you can declare property like `pointer: number`, but in 2.7 and later, you should declare it like `pointer?: number` or `pointer!: number`, because start with Typescript version 2.7, there is a new tsconfig option of **strictPropertyInitialization**, it will checks to esure each instance property of a class gets initialized, look at the [details](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html). Search quickly with keywords: **Strict Class Initialization**
```

Comment:
```suggestion
Notice, if your TypeScript version before 2.7, you can declare property like `pointer: number`, but in 2.7 and later, you should declare it like `pointer?: number` or `pointer!: number`, because starting with Typescript version 2.7, there is a new tsconfig option called **strictPropertyInitialization**, which will check to ensure each instance property of a class gets initialized. For more information, refer to the [release notes](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-7.html#strict-class-initialization).
```

Reply:
I fixed a few typos and changed the wording to make it more clear

Reply:
@c690554125 Let me know when this is ready for re-review